x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/TransportDeprecationInfoAction.java

```diff
@@ -128,6 +144,15 @@ protected final void masterOperation(
         final ActionListener<DeprecationInfoAction.Response> listener
     ) {
         PrecomputedData precomputedData = new PrecomputedData();
+        DeprecationIssue lowWatermarkIssue = checkDiskLowWatermark(
```

Comment:
That must be what I was looking at earlier, lowWatermarkIssue could be just a final field in `PrecomputedData`. Every `SetOnce` we can get rid of makes small pieces of code easier to understand, so imho it's worth it